content/blog/tune-0-1-1/index.Rmd

```diff
@@ -0,0 +1,255 @@
+rf_param <- parameters(rf_mod)
+rf_param
+```
+
+There are parameter objects for these (and they keep their original names). You can adjust the ranges and values for these parameters using the `update()` function as you would for others. To see their underlying functions: 
+
+```{r show-param}
+rf_param$object[[2]]
+```
+
+From here, the standard tools in the tune package can be used. You can use one of the `grid_*()` functions to create a grid of values, let `tune_grid()` create a set for you, or use Bayesian optimization to find appropriate values sequentially. 
+
+The new `autoplot()` method can also be used to produce nice visualizations of the relationship between performance and the parameters. 
+
+```{r rf-tune, dev = 'svg', dev.args = list(bg = "transparent"), fig.align='center', fig.height=4}
+set.seed(4976)
+ranger_params <-
+  rf_mod %>%
+  tune_grid(
+    ridership ~ Harlem + Archer_35th,
+    resamples = data_folds,
+    grid = 10,
+    control = ctrl
+  )
+
+autoplot(ranger_params, metric = "rmse") + 
+  theme(legend.position = "top")
+```
+
+Note that the tuning parameter labels (i.e. "Gain Penalization" instead of "`regularization.factor`") are used. 
+
+I'm sure that we missed someone's favorite engine-specific parameter so please put in a [GitHub issue for dials](https://github.com/tidymodels/dials/issues) to let us know. 
+
+## `.config` columns
+
+When model tuning is conducted, the tune package now saves a new column in the outputs called `.config`. This column is a qualitative identification column for unique tuning parameter combinations. It often reflects what is being tuned. A value of `.config = "Recipe1_Model3"` indicates that the first recipe tuning parameter set is being evaluated in conjunction with the third set of model parameters. Here's an example from the random forest model that we just fit: 
+
+```{r rf-config}
+ranger_params %>% 
+  collect_metrics() %>% 
+  # get the unique tuning parameter combinations:
+  select(min_n, regularization.factor, regularization.usedepth, .config) %>% 
+  distinct()
+```
+
+## Other changes
+
+* `conf_mat_resampled()` computes the average confusion matrix across resampling statistics for a single model.  
+
+* `show_best()`, and the `select_*()` functions will now use the first metric in the metric set if no metric is supplied. 
+
+* `filter_parameters()` can trim the `.metrics` column of unwanted results (as well as columns `.predictions` and `.extracts`) from `tune_*` objects. 
+
+* If a grid is given, parameters do not need to be finalized to be used in the `tune_*()` functions. 
+
+* Added a `save_workflow` argument to `control_*` functions that will result in the workflow object used to carry out tuning/fitting (regardless of whether a formula or recipe was given as input to the function) to be appended to the resulting `tune_results` object in a `workflow` attribute. The new `.get_tune_workflow()` function can be used to access the workflow.
```

Comment:
Do you want to clarify at all that this will probably just be used internally? That or you could just leave this bullet out entirely if you don't feel that users would ever need this

Reply:
I'll remove it. I doubt anyone would use it.